### PR TITLE
Report zookeeper monitoring statistics in configserver

### PR DESCRIPTION
**Pull request by ollivir**
Metric names to be verified (hence, DNM tag).

### CODE REVIEW

File: configserver/src/main/java/com/yahoo/vespa/config/server/monitoring/ZKMetricUpdater.java [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+    private final int CONNECTION_TIMEOUT_MS = 500;
+    private final int WRITE_TIMEOUT_MS = 250;
+    private final int READ_TIMEOUT_MS = 500;
+
+    private AtomicReference<Map<String, Long>> zkMetrics = new AtomicReference<>(new HashMap<>());
+    private final Timer timer = new Timer();
+    private final int zkPort;
+
+    public ZKMetricUpdater(Metrics metrics, ZookeeperServerConfig zkServerConfig, HealthMonitorConfig healthMonitorConfig) {
+        this(zkServerConfig, 4500, (long) (healthMonitorConfig.snapshot_interval() * 1000));
```
Comment (hmusum) [inline configserver/src/main/java/com/yahoo/vespa/config/server/monitoring/ZKMetricUpdater.java]: metrics seems to be unused. Remove?

Comment (hmusum) [reply configserver/src/main/java/com/yahoo/vespa/config/server/monitoring/ZKMetricUpdater.java]: Make 4500 a constant? Why 4500, btw? Make it dependent on healthMonitorConfig.snapshot_interval() instead?

**Review by hmusum — APPROVED** [review]
Looks good to me, two minor comments, feel free to merge

File: configserver/src/main/java/com/yahoo/vespa/config/server/monitoring/ZKMetricUpdater.java (hunk 1 above)
Comment (ollivir) [reply configserver/src/main/java/com/yahoo/vespa/config/server/monitoring/ZKMetricUpdater.java]: `metrics` is a leftover from how the code worked a bit earlier, will remove before merge. 4500 reflects a similar constant in `Metrics` where the initial delay is 5000. The idea is to try and trigger this one slightly before. Of course drift might make that a pointless exercise anyway.

**Comment by hmusum** [comment]
👍

**Pull request merged**